### PR TITLE
#1315 right toolbar redesign

### PR DESCRIPTION
**Pull request by Mariia-Katkova**
Changed toolbar view for desktop, moved button classes to atom component, added some styles according to the design

### CODE REVIEW

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.module.less [hunk 1]
```diff
@@ -19,6 +19,12 @@
 .root {
   .toolbar();
   .vertical-toolbar();
+  position: relative;
+  top: 0;
+  right: 60px;
+  margin-top: 20px;
+  margin-right: 20px;
+  z-index: 10;
```
Comment (karen-sarkisyan) [inline packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.module.less:28]: Note that this toolbar is nested within a rightmost column of a grid defined in `App.module.less`. Having a `margin-right` causes grid column to bulge, but `right` makes the toolbar render outside of its grid-area:
image

As long as we don't need that column anymore, maybe it'd be better to remove right column from root grid at all, and just position the Elements panel `absolute`ly?

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.tsx [hunk 2]
```diff
@@ -73,25 +77,20 @@ const RightToolbar = (props: Props) => {
             onAction={onAction}
           />
           <AtomsList atoms={freqAtoms} active={active} onAction={onAction} />
-        </Group>
-
-        <Group>
-          <div ref={sizeRef}>
+          <div className={classes.toolbarButton} ref={sizeRef}>
             <ToolbarGroupItem id="period-table" {...rest} />
```
[the remark this answers is not in the document]
Comment (Mariia-Katkova) [reply packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.tsx:81]: changed the icon

File: packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.module.less (hunk 1 above)
Comment (Mariia-Katkova) [reply packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/RightToolbar.module.less]: removed the right column

File: packages/ketcher-react/src/script/ui/component/view/Atom/Atom.jsx [hunk 3]
```diff
@@ -14,11 +14,16 @@
  * limitations under the License.
  ***************************************************************************/
 
-function Atom({ el, shortcut, className, ...props }) {
+import classes from '../../../views/toolbars/ToolbarGroupItem/ActionButton/ActionButton.module.less'
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/component/view/Atom/Atom.jsx:17]: All styles related to Atom component should de defined inside Atom.module.less. Please do  not reuse styles of ActionButton component

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
see comments

**Pull request closed without merging**